v2/config/samples/compute/v1beta20220301_image.yaml

```diff
@@ -0,0 +1,29 @@
+apiVersion: compute.azure.com/v1beta20220301
+kind: Image
+metadata:
+  name: aso-sample-image
+  namespace: default
+spec:
+    extendedlocation: null
+    hypervgeneration: V2
+    location: westus2
+    owner:
+        name: aso-sample-rg
+    sourcevirtualmachine: null
+    storageprofile:
+        datadisks: []
+        osdisk:
+            bloburi: null
+            caching: null
+            diskencryptionset: null
```

Comment:
Is this casing really correct? Shouldn't it be `diskEncryptionSet`?

Reply:
I agree, casing does not look alright for most of the fields in the sample.